src/main/java/org/opentripplanner/routing/vehicle_parking/VehicleParkingService.java

```diff
@@ -78,4 +78,12 @@ public Stream<VehicleParking> getVehicleParkings() {
   public ImmutableListMultimap<VehicleParkingGroup, VehicleParking> getVehicleParkingGroups() {
     return vehicleParkingGroups;
   }
+
+  public boolean hasBikeParkingLots() {
```

Comment:
We should be consistent with the naming here. In this file we use, Parks, Parkings and ParkingLots. Also, I don't know if we should use Parkings or just Parking because parking is the plural of parking but just parking doesn't really indicate if it's plural or not.